packages/rum-core/src/domain/view/viewCollection.ts

```diff
@@ -132,6 +132,6 @@ function processViewUpdate(
   }
 }
 
-function discardNegativeDuration(duration: ServerDuration | undefined): ServerDuration | undefined {
+export function discardNegativeDuration(duration: ServerDuration | undefined): ServerDuration | undefined {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Since it is no longer specific to views, you could move the function up in a new `browser/timeUtils.ts` for instance.

Reply:
Or maybe in `domain/discardNegativeDuration.ts` :)